src/inverseRobot.js

```diff
@@ -24,7 +24,44 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+
+  for (const key in robot) {
+    if (robot[key] in inverse) {
```

Comment:
```suggestion
    if (inverse.hasOwnProperty(robot[key])) {
```
This check is better. [Here is why.](https://thisthat.dev/has-own-property-vs-in/)